topcoffea/modules/createJSON.py

```diff
@@ -85,7 +85,9 @@ def main():
         dicFiles = GetDatasetFromDAS(dataset, nFiles, options='file', withRedirector=prefix)
         files = [f[len(prefix):] for f in dicFiles['files']]
         files_with_prefix = dicFiles['files']
-
+        if 'root://cms-xrd-global.cern.ch//store/mc/Run3Summer22NanoAODv12/ZZZ_TuneCP5_13p6TeV_amcatnlo-pythia8/NANOAODSIM/130X_mcRun3_2022_realistic_v5-v2/50000/7c4f3eb2-3c7e-4c21-98ed-c1892bb3a057.root' in files_with_prefix:
+            print("ZZZ sample")
+            files_with_prefix.remove('root://cms-xrd-global.cern.ch//store/mc/Run3Summer22NanoAODv12/ZZZ_TuneCP5_13p6TeV_amcatnlo-pythia8/NANOAODSIM/130X_mcRun3_2022_realistic_v5-v2/50000/7c4f3eb2-3c7e-4c21-98ed-c1892bb3a057.root')
```

Comment:
Are these lines for a bad file?

Reply:
Yes, the one missing an LHE branch

Reply:
Could you explain the handling here? So you are skipping a file? I am not sure we want to hard code this like this. This module is used by multiple analyses, and it's supposed to be fairly general. So this relatively silent skipping of a very particular root file is maybe not exactly how we want to handle it. 
Maybe would be better to pass an option to skip a file, or to just e.g. make this change locally if you don't want the file.

Also, what is wrong with the file?

Reply:
According to GEN contacts, this is the issue. Given that, we can remove these lines, not using the ZZZ sample for now.
What do you think @bryates @kmohrman ?

<img width="1024" alt="Screenshot 2024-11-05 at 10 05 42" src="https://github.com/user-attachments/assets/9a05d4de-c656-4104-ab52-3c610a11a9aa">

Reply:
I guess this is fine for a temporary solution. The email from the GEN contacts makes it sound like it _might_ be fixable, so I opened an issue to remind us to check on this later.